egs/uw3/v1/local/chain/align_nnet3_lats.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
No need for this or the chain_ali recipe because the improvement from chain alignments is too small.